vignettes/dust.Rmd

```diff
@@ -47,21 +47,21 @@ There are two main parts here; the class (`walk`) and an interface function (`du
 The class is what does most of the work. It is important that this does not use anything in the R api as it may be called in parallel. Therefore use only C++ standard library types.  It has to provide every public type or method as the example above:
 
 * the type `real_t` has to exist and indicate what sort of numeric type you use for "real numbers". Typically this will be `double`,but if you need a different type (longer or shorter) you can use it here.
-* the type `init_t` is whatever internal data your model needs, and an object of this type will be passed to your constructor (see `dust_data` below)
-* the constructor must take only one argument, being `const init_t&`. You can do what you want in the constructor - here we just copy the data into the object (we use the private member `data_` here but this can be anything you want)
+* the type `init_t` is whatever internal data your model needs (most likely its parameters), and an object of this type will be passed to your constructor (see `dust_pars` below)
+* the constructor must take only one argument, being `const init_t&`. You can do what you want in the constructor - here we just copy the pars into the object (we use the private member `pars_` here but this can be anything you want)
```

Comment:
```suggestion
* the constructor must take only one argument, being `const init_t&`. You can do what you want in the constructor - here we just copy the parameters into the object with the `pars` argument (we use the private member `pars_` here but this can be anything you want)
```